plenum/test/view_change/test_view_change_with_instance_change_lost_due_to_restarts.py

```diff
@@ -0,0 +1,42 @@
+import pytest
+
+from plenum.test.helper import freshness, waitForViewChange
+from plenum.test.node_request.helper import sdk_ensure_pool_functional
+from plenum.test.restart.helper import restart_nodes
+from plenum.test.test_node import ensureElectionsDone
+
+FRESHNESS_TIMEOUT = 20
+
+
+@pytest.fixture(scope="module")
+def tconf(tconf):
+    with freshness(tconf, enabled=True, timeout=FRESHNESS_TIMEOUT):
+        yield tconf
+
+
+def test_view_change_with_instance_change_lost_due_to_restarts(looper, txnPoolNodeSet,
+                                                               sdk_pool_handle,
+                                                               sdk_wallet_client,
+                                                               tconf, tdir, allPluginsPath):
+    """
+    1. some_nodes (Beta and Gamma) send InstanceChange for all nodes.
+    2. Restart other_nodes (Gamma and Delta)
+    3. last_node (Delta) send InstanceChange for all nodes.
+    4. Ensure elections done and pool is functional
+    """
+    current_view_no = txnPoolNodeSet[0].viewNo
+    some_nodes = txnPoolNodeSet[1:3]
+    other_nodes = txnPoolNodeSet[2:4]
+
+    for n in some_nodes:
+        n.view_changer.on_master_degradation()
+        looper.runFor(0.2)
```

Comment:
Why do we need this run? Should we check instead that other nodes received 2 IC messages?

Reply:
Yes, check for received IC messages would be more clean, current implementation is just a quick hack around.

Reply:
Fixed